blender_render.ipynb

```diff
@@ -40,7 +40,7 @@
       },
       "outputs": [],
       "source": [
-        "blender_version = '3.2.2' #@param ['2.79b', '2.80rc3', '2.81a', '2.82a', '2.83.20', '2.90.1', '2.91.2', '2.92.0', '2.93.10', '3.0.1', '3.1.2', '3.2.2'] {allow-input: false}\n",
+        "blender_version = '3.2.2' #@param ['2.79b', '2.80rc3', '2.81a', '2.82a', '2.83.20', '2.90.1', '2.91.2', '2.92.0', '2.93.10', '3.0.1', '3.1.2', '3.2.2', '3.3.0'] {allow-input: false}\n",
```

Comment:
Can you change the default blender version to 3.3.0?

Reply:
Sure